src/java.desktop/share/classes/com/sun/java/swing/plaf/gtk/GTKLookAndFeel.java

```diff
@@ -548,7 +548,7 @@ public Object createValue(UIDefaults table) {
             "CheckBoxMenuItem.alignAcceleratorText", Boolean.FALSE,
 
 
-            "ColorChooser.showPreviewPanelText", Boolean.FALSE,
+            "ColorChooser.showPreviewPanelText", Boolean.TRUE,
```

Comment:
Is there any preview panel in GTK native app? If you could not find anything in native, then let's keep it as false and make the test prevent from running for GTK L&F..
It's like SplitPane.supportsOneTouchButton property which is false for gtk and we restrict running for GTK L&F...

Reply:
> Is there any preview panel in GTK native app? If you could not find anything in native, then let's keep it as false and make the test prevent from running for GTK L&F..

I couldn't find any application in native side for color chooser dialog. However I found this [GTKColorChooserDialog GTK3.24 docs](https://devdocs.io/gtk~3.24/gtkcolorchooserdialog) widget link and it seems the GTKColorDialog is totally different than what we have in jdk library.